src/core/utils/index.js

```diff
@@ -376,3 +376,10 @@ export const findFileForPlatform = ({ userAgentInfo, platformFiles }) => {
   const platform = agentOsName && userAgentOSToPlatform[agentOsName];
   return (platform && platformFiles[platform]) || platformFiles[OS_ALL];
 };
+
+export const getLocalizedTextWithLinkParts = (localizedTextWithLink) => {
+  invariant(localizedTextWithLink, 'localizedTextWithLink is required');
+  return localizedTextWithLink
```

Comment:
Please see my comment above about how I think we could actually move even more of the logic into this function. Also, I think this could return something in a bit friendlier format - so rather than returning an array, it could return an object with properties like `beforeLinkText`, `afterLinkText` and `innerLinkText`.

Reply:
thanks @bobsilverberg. I do like the idea of returning an object with these key names.